contract/src/lib.rs

```diff
@@ -1,20 +1,18 @@
 use near_sdk::collections::{LookupMap, UnorderedSet};
 use near_sdk::json_types::Base64VecU8;
 use near_sdk::{
-    borsh::{self, BorshDeserialize, BorshSerialize},
-    ext_contract, log,
-    serde::{Deserialize, Serialize},
-    Balance, PanicOnDefault, Promise, PromiseResult,
+    ext_contract, log, Allowance, NearToken, PanicOnDefault, Promise, PromiseResult
 };
-use near_sdk::{env, near_bindgen, AccountId, PublicKey};
+use near_sdk::{env, near, AccountId, PublicKey};
 use near_sdk::{serde_json, Gas};
 use std::convert::TryFrom;
+use std::str::FromStr;
 
 // 5 Ⓝ in yoctoNEAR
 const PRIZE_AMOUNT: u128 = 5_000_000_000_000_000_000_000_000;
```

Comment:
This should probably be of type `NearToken`.

Reply:
fixed